src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -253,7 +254,7 @@ async def Generate(
                 prompt=req.text,
                 prompt_token_ids=input_ids,
             )
-            kwargs = {}
+            kwargs = adapter_kwargs
```

Comment:
I guess we can just get rid of this>

Reply:
You mean get rid of separate `adapter_kwargs` var?

Reply:
yeah